cpg_infra/billing_aggregator/aggregate/gcp.py

```diff
@@ -128,15 +128,14 @@ def get_billing_data(start: datetime, end: datetime):
         FROM `{utils.GCP_BILLING_BQ_TABLE}`
         WHERE export_time >= @start
             AND export_time <= @end
-            AND project.id <> @seqr_project_id
+            AND project.id NOT IN UNNEST(@projects)
```

Comment:
`projects` isn't defined in the ArrayQueryParameter below, I'm also a little surprised you need to UNNEST projects.